sgt-back-end/server.js

```diff
@@ -137,9 +116,10 @@ app.post('/api/grades', async (req, res) => {
     const params = [course, name, score];
     const result = await db.query(sql, params);
     const grade = result.rows[0];
-    if (grade) {
-      res.json(grade);
+    if (!grade) {
+      return;
```

Comment:
You need to send a result before returning. Here is where you would tell the user you can't find the grade ID.